fission-web-server/library/Fission/Web/Server/Email/Error.hs

```diff
@@ -0,0 +1,20 @@
+module Fission.Web.Server.Email.Error (CouldNotSend(..)) where
+
+import           Servant.Client
+import           Servant.Server
+
+import           Fission.Prelude
+
+import           Fission.Web.Server.Error.Class
+
+newtype CouldNotSend = CouldNotSend ClientError
+  deriving ( Show
+           , Eq
+           )
```

Comment:
Also pro-tip: do you know about `deriving newtype`? Doesn't make a huge difference here, but mainly an FYI